R/callouts.R

```diff
@@ -38,10 +38,11 @@ callouts.gsplot <- function(object, ..., side=c(1,2)){
 #' @param labels text to be added to callout
 #' @param length relative (percentage of window width and height) distance for callout
 #' @param angle callout line angle
+#' @param cex passed to text for font size formatting
 #' 
 #' @rdname callouts
 #' @export
-callouts.default <- function(x, y=NULL, labels=NA, length=0.1, angle='auto', cex=1, ...){
+callouts.default <- function(x, y=NULL, labels=NA, length=0.1, angle='auto', cex=par()$cex, ...){
```

Comment:
is this the `par()` at load time, or the `par()` at runtime?  This might be an issue if we change our `par()` scoping.